client/interfaces.go

```diff
@@ -228,6 +228,8 @@ type InstanceServer interface {
 	DeleteNetwork(name string) (err error)
 
 	// Network ACL functions ("network_acl" API extension)
+	GetNetworkACLs() (acls []api.NetworkACL, err error)
+	GetNetworkACL(name string) (acl *api.NetworkACL, ETag string, err error)
```

Comment:
This one was meant for the next commit I think.

Reply:
Fixed.